src/resource/file.ts

```diff
@@ -135,6 +148,15 @@ type SaveFileOptions = WriteFileOptions & {
  * Saves a file in a folder associated with the given URL. The final filename may or may
  * not be the given `slug`.
  *
+ * If you know the [[https://developer.mozilla.org/en-US/docs/Glossary/MIME_type | media type]]
```

Comment:
Are you sure this syntax works (and if so, where is it defined)? In the end it's going to be parsed by Sphinx's markdown parser, so I know the following syntax should work - though if the current syntax works, that's fine too of course.

This also applies to the other links in this PR.

```suggestion
 * If you know the [media type](https://developer.mozilla.org/en-US/docs/Glossary/MIME_type)
```

Reply:
Ah of course I can just check that it looks fine: https://lit-pod-e7sms1j08.vercel.app/modules/resource_file.html#overwritefile

(Although the link to `[[options]]` does not. Anyway, I'm in the process of fixing up this PR, so I'll fix it.)